frontend/src/pages/LearningCourse/CourseInfoToggle/CourseInfoToggle.tsx

```diff
@@ -1,5 +1,7 @@
 import { FC } from 'react';
 
+import { ButtonLabels } from 'constants/ButtonLabels';
+
 import { CourseInfoToggleButton, ExpandLessIcon, ExpandMoreIcon } from './styled';
 
 interface IProps {
```

Comment:
Do we need to change Interface name sccording to component name

Reply:
I think, we'll have an additional ticket for this improvement, but I'll fix this file interface name